src/translations/locale/locale.en-US.json

```diff
@@ -0,0 +1,4 @@
+{
+  "validation.email.invalid": "Email is not correct",
+  "validation.required": "Field is required"
+}
```

Comment:
Sorry, typo :). "It doesn't work with the validation mechanism out of the box". In `er-common-components` there are some built in validators, created specially for `redux-form`. `redux-form` for works with `string` validation texts. So if your validator returns "You have an error on this field" and not `undefined`, it will show that error and mark the field as with error.

A sample of validator in `er-common-compoents/src/helpers/validation`:
```
export function required(value) {
  if (isEmpty(value)) {
    return {
      key: 'validation.required',
      text: 'Field is required',
    };
  }
  return null;
}
```

So `mapValidationMessagesToLocale` will map the result of the `required(value).key` validator to the translated text using `react-intl:formatMessage` function. Eg: `formatMessage({id: required(value).key})`.

So in the end for form validation all we need to have, are the keys returned by `er-common-compoents/src/helpers/validation` inside our translation keys, so we show translated error messages.

For other things, like labels, link texts etc, we should use directly `<FormattedMessage />` component.

Hope this clarifies how it works :). 

I will add the rest of the translations today.